### PR TITLE
Re-added view_metabolic_model_details that apparently didn't make the…

### PR DESCRIPTION
**Pull request by briehl**
… last cut.

There may be others that come up later.

### CODE REVIEW

File: methods/view_metabolic_model_details/display.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#
+# Define basic display information
+#
+name     : View Metabolic Model Details
+subtitle : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```
Comment (nlharris) [inline methods/view_metabolic_model_details/display.yaml]: should say "Bring up a detailed view of a metabolic model". (Not sure how to edit in the PR.)

File: methods/view_metabolic_model_details/display.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+tooltip  : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```
Comment (nlharris) [inline methods/view_metabolic_model_details/display.yaml]: Same here.

File: methods/view_metabolic_model_details/display.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        ui-name : |
+            Metabolic Model
+        short-hint : |
+            the metabolic model to view [7.1]
+        long-hint  : |
+            the metabolic model to view [7.1]
+
+
+description : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```
Comment (nlharris) [inline methods/view_metabolic_model_details/display.yaml]: should say "Bring up a detailed view of a metabolic model".

**Comment by briehl** [comment]
Edited and changed, and removed the [7] and [7.1] numbers, too.
That method won't show up in the methods panel, anyway, but it's needed for the drag and drop viewer to work.

**Pull request merged**